resources/qml/Cura.qml

```diff
@@ -359,157 +143,199 @@ UM.MainWindow
                 }
             }
 
-            JobSpecs
+            Loader
             {
-                id: jobSpecs
+                // The stage menu is, as the name implies, a menu that is defined by the active stage.
+                // Note that this menu does not need to be set at all! It's perfectly acceptable to have a stage
+                // without this menu!
+                id: stageMenu
+
                 anchors
                 {
-                    bottom: parent.bottom;
-                    right: sidebar.left;
-                    bottomMargin: UM.Theme.getSize("default_margin").height;
-                    rightMargin: UM.Theme.getSize("default_margin").width;
+                    left: parent.left
+                    right: parent.right
+                    top: parent.top
+                    margins: UM.Theme.getSize("default_margin").height
                 }
+
+                height: 50
```

Comment:
Should be themeable, as well as scale with the pixel size.

Reply:
Done in 1cc7e0e586604ddf16b2931e1bcffbadb53c7f80